asf_lifetime_cost_model/config/scenarios.py

```diff
@@ -0,0 +1,52 @@
+"""Scenarios configuration file.
+
+Dictionary with a list of pre-set scenarios and respective arguments used
+to calculate the lifetime cost of different heating systems.
+"""
+
+scenarios = {
+    "baseline_A": {
+        "name": "Baseline",
+        "ashp_scop": "reference",
+        "ashp_subsidy": "Slow stepdown",
+        "ashp_market_cost_change": "moderate",
+        "purchasing_with_loans": True,
+        "interest_rate": 0.05,
```

Comment:
I think it would be good to link this input to what is in base.yaml, rather than hard-coding. I'm not sure how this could work without importing `config` and doing

```suggestion
        "interest_rate": config.get("loan_interest_rate_default")
```


I do however anticipate the team will want to explore different interest rate options, so I think that it might be better to replace `loan_interest_rate_default` in base.yaml with `loan_interest_rate_options`, like:

```
loan_interest_rate_options: [
    { "name": "default", "interest_rate": 0.05 }, # 5%
    { "name": "low", "interest_rate": 0.025 }, # 2.5%
    { "name": "high", "interest_rate": 0.1 }, # 10%
  ]
```

or any other format you think would be best! the low and high values I've suggested are just ideas, we can also ask the policy team what rates they might want to consider.

if we change to a dictionary format then this line could be

```suggestion
        "interest_rate": "default",
```

Reply:
I think it's a good idea to get a sense from them on what they want. Either way, I think we would always set a default value and then allow people to change it to whatever they want.

Reply:
Also, yes about not hardcoding it. Just changed for it to call the config!